### PR TITLE
Lab4

### PR DESCRIPTION
**Pull request by bkmceh**
@danmory
@ezio-42
U r welcome

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+# Include tfplan files to ignore the plan output of command: terraform plan -out=tfplan
+# example: *tfplan*
+
+# Ignore CLI configuration files
+.terraformrc
+terraform.rc
+**/*.pub
+**/keys
+**/*.json
```
Comment (danmory) [inline .gitignore:173]: ignore all json files? Is it a good idea? Maybe better to use more specific pattern?

File: app_python/test_show_time_app.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+@pytest.fixture
+def client():
+    app.config["TESTING"] = True
+    with app.test_client() as client:
+        yield client
+
+
+def test_time(client):
+    r = client.get("/")
+    assert r.status_code == 200
```
Comment (danmory) [inline app_python/test_show_time_app.py:17]: still no EOF

File: terraform/yandex-cloud-terraform/variables.tf [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+variable "app_server_ssh_key" {
+  description = "Path to public key"
+  type        = string
+  default     = "keys.pub"
+}
+
+variable "image_id_name" {
+  description = "Image id"
```
Comment (danmory) [inline terraform/yandex-cloud-terraform/variables.tf:20]: description does not tell what variable is about

File: terraform/yandex-cloud-terraform/variables.tf [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  default     = 2
+}
+
+variable "app_server_ssh_key" {
+  description = "Path to public key"
+  type        = string
+  default     = "keys.pub"
+}
+
+variable "image_id_name" {
```
Comment (danmory) [inline terraform/yandex-cloud-terraform/variables.tf:19]: name of variable also strange: id or name?

**Comment by ezio-42** [comment]
LGTM, but next time push earlier or you will have to fix possible issues in 10 minutes before deadline

**Pull request closed without merging**